css/amp-story-prestyle.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * Styles that can be preapplied to amp-story[standalone] documents to layout
+ * the story before the JS loads.
+*/
+
+html.i-amphtml-singledoc.i-amphtml-standalone body, body,
+html.i-amphtml-singledoc.i-amphtml-standalone, html {
+  animation:none !important;
+  height: 100% !important;
+  width: 100% !important;
+  margin: 0;
+  overflow: hidden;
+}
+
+.i-amphtml-notbuilt {
+  color: initial !important;
+}
+.i-amphtml-notbuilt > * {
+  display: initial;
+}
```

Comment:
This is OK for this boilerplate PR but as we discussed, in the next PR this should be a `@import 'amp-story.css'`

Reply:
Yes, I think some of these styles will be needed (they are not in `amp-story.css`) but on the next PR I'll import the CSS and deal with it.